tendrl/node_agent/discovery/sds/manager.py

```diff
@@ -0,0 +1,50 @@
+import importlib
+import inspect
+import logging
+import os
+import sys
+
+from tendrl.node_agent.discovery.sds.discover_sds_plugin \
+    import DiscoverSDSPlugin
+
+LOG = logging.getLogger(__name__)
+
+
+class SDSDiscoveryManager(object):
+    def __init__(self):
+        try:
+            self.load_plugins()
+        except (SyntaxError, ValueError, ImportError) as ex:
+            LOG.error(
+                'Failed to load the sds discovery plugins. Error %s' %
+                ex, exc_info=True
+            )
+            raise ex
```

Comment:
in init method looks like general norm in python to raise ValueError, can you raise that here?